README.md

```diff
@@ -1 +1,204 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Fully functional Backend]**
+- **[Fully functional Frontend]**
+- **[Dynamic routing]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming soon...]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+Clone this repository to your desired folder:
+
+```bash
+$ git clone https://github.com/Mylo16/Budget-App.git
+$ cd Budget-App
+```
+
+
+
+### Prerequisites
+
+In order to work on this project, you need to have the following dependencies installed:
+
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Postgresql](https://www.postgresql.org/)
+- [Node.js](https://nodejs.org/en/)
+- [Rails](https://rubyonrails.org/)
+
+
+
+### Install
+
+Install this project with:
+
+```bash
+$ bundle install
+```
+
+To get all the gems required for the project
+
+Then, run
+
+```bash
+$ npm install
+```
+
+To install packages such as style linters
+
+To check linters locally, use
+
+```
+bash
+$ rubocop
+$ npx stylelint "**/*.{css,scss}"
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+```
+cd my folder
+git clone  https://github.com/Mylo16/Budget-App.git
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+  ```sh
+    rspec ./spec/integration
+    rspec ./spec/requests
+    rspec ./spec/models
+  ```
+
+## Author <a name="author"></a>
+
+👤 **Eric Antwi**
+
+- GitHub: [@Mylo16](https://github.com/Mylo16)
+- Twitter: [Eric](https://twitter.com/EricAntwi384)
+- LinkedIn: [Eric](https://www.linkedin.com/in/ericopokuantwi)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Ship code to production]**
+- [ ] **[Build Desktop version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please express your appreciation on LinkedIn.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all those who helped me throughout this project.
+
```

Comment:
- [ ] As per the requirement for this capstone, please give an appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)